xc/xenops_server_xen.ml

```diff
@@ -2198,8 +2198,11 @@ module VUSB = struct
            let privileged = is_privileged vm in
            Device.Vusb.vusb_unplug ~xs ~privileged ~domid:frontend_domid ~id:(snd vusb.Vusb.id) ~hostbus:vusb.Vusb.hostbus ~hostport:vusb.Vusb.hostport
         ) Newest vm
-    with (Does_not_exist(_,_)) ->
+    with
+    | (Does_not_exist(_,_)) ->
       debug "VM = %s; VUSB = %s; Ignoring missing domain" vm (id_of vusb)
+    | (Device_not_connected) ->
+      debug "VM = %s; VUSB = %s; Ignoring missing device" vm (id_of vusb)
```

Comment:
This can't have the same log message as above because this is a different reason.

Reply:
I think "Ignoring missing device" is different from above.

Reply:
You are right - I misread this.